frontend/src/wsApi.ts

```diff
@@ -46,11 +59,11 @@ export const useWsApi = (url) => {
     });
 
     socket.current.addEventListener("message", (data) => {
-      const event = JSON.parse(data.data);
+      const event: DataInterface = JSON.parse(data.data);
```

Comment:
There is no validation here right? Just typecasting?

Reply:
Correct